apps/base.py

```diff
@@ -13,8 +13,7 @@
 import terminalio
 from adafruit_display_shapes.rect import Rect
 from adafruit_display_text import label
-
-from app_pad import AppPad, DoubleTapEvent, EncoderEvent, EncoderButtonEvent, KeyEvent
+from app_pad import AppPad, DoubleTapEvent, EncoderButtonEvent, EncoderEvent, KeyEvent
```

Comment:
This file should also be deleted, correct?

Reply:
You are quite right, missed it!
Removed.